### PR TITLE
Loads BF Item Profile and tests for resourceTemplates in BIBFRAME 2.0…

### LINKED ISSUE #49: Test importing a profile
- Visit http://localhost:8000/#/profile/create
- Click on Import button: `href="#styledModal"`
- Expect some html elements: 
```
<div class="modal-content">
            <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">x</button>
                <h3 class="modal-title">Import Data</h3>
            </div>
            <div class="modal-body" sss-drop="">
                <form action="./dummy.html" class="dropzone dz-clickable ng-pristine ng-valid" id="dropzone">
                    <div class="dz-default dz-message">
                        <span>
                            <span class="bigger-150 bolder">
                                <i class="ace-icon fa fa-caret-right red"></i>
                                Drop file
                            </span>
                            to upload
                            <br>
                            <i ng-file-select="onFileSelect($files)" class="upload-icon ace-icon fa fa-cloud-upload blue fa-3x" style="overflow: hidden;"><input type="file" ng-file-select="onFileSelect($files)" class="upload-icon ace-icon fa fa-cloud-upload blue fa-3x" __wrapper_for_parent_="true" style="width: 1px; height: 1px; opacity: 0; position: absolute; padding: 0px; margin: 0px; overflow: hidden;"></i>
                        </span>
                    </div>
                </form>
            </div>
            <div class="modal-footer">
                <button id="importCloseButton" type="button" class="btn btn-default" data-dismiss="modal">Close</button>
            </div>
        </div>
```

- Simulate file drop or selection
  - this may help: https://reactjsnews.com/testing-drag-and-drop-components-in-react-js

- Expect form element with:  `name="profileForm"`

- Expect with name of profile (e.g.):
```
<h3 class="portlet-title">
        <u class="ng-binding">BIBFRAME 2.0 Agents</u>
    </h3>
```

- Expect spans with (e.g. depending on profile uploaded)
```
<span popover="URI: http://www.loc.gov/mads/rdf/v1#PersonalName" popover-title="Resource ID: profile:bf2:Agent:Person" popover-trigger="mouseenter" popover-placement="right" class="ng-scope ng-binding">Person</span>
```

### PR DESCRIPTION
**Pull request by jermnelson**
Loads BF Item Profile and tests for resourceTemplates labels in
a loaded BIBFRAME 2.0 Item Profile

Closes #49

### CODE REVIEW

File: source/__tests__/integration/profile-create.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    await page.goto('http://127.0.0.1:8000/#/profile/create')
+  })
+
+  describe('Import Data Modal Upload Dialog', () => {
+
+    beforeAll(async () => {
+      await expect(page).toClick('a', { text: 'Import'})
+    })
+
+    it('click the import button and display dialog', async () => {
```
Comment (ndushay) [inline source/__tests__/integration/profile-create.test.js]: Is this better worded as "displays Import Data dialog" as the clicking is happening in beforeAll?

File: source/__tests__/integration/profile-create.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    it('uploads a local BIBFRAME 2.0 Item Profile and checks for loaded Profile', async () => {
+      const bf_item_location = path.join(__dirname, "..", "__fixtures__", 'item.json')
+
+      await expect(page).toUploadFile(
+        'input[type="file"]',
+        bf_item_location,
+      )
+
+      await page.waitFor(1000) // Waits for the Profile to load
+      await expect(page).toMatch(/BIBFRAME 2.0 Item/)
```
Comment (ndushay) [inline source/__tests__/integration/profile-create.test.js]: I like these matchers as they capture what we'd look for specific to the Item Profile

File: .gitignore [hunk 3]
```diff
@@ -6,3 +6,4 @@ documentation/ngdocs
 junit.xml
 node_modules/
 static-analysis
+junit.xml
```
Comment (ndushay) [inline .gitignore]: junit.xml is already there - line 6

**Comment by jgreben** [comment]
Guess I was too hasty to merge...

**Comment by jgreben** [comment]
@ndushay @jermnelson I will make some fixes and submit a new PR.

**Pull request merged**